lldb/test/API/lang/swift/xcode_sdk/TestSwiftXcodeSDK.py

```diff
@@ -36,3 +33,32 @@ def test(self):
                 found += 1
         self.assertEqual(in_expr_log, 1)
         self.assertEqual(found, 1)
+
+    @swiftTest
+    @skipUnlessDarwin
+    @skipIfDarwinEmbedded
+    def test_override(self):
+        """Test that we can override the Xcode SDK using the target setting."""
+        self.build()
+        log = self.getBuildArtifact("types.log")
+        self.expect("log enable lldb types -f " + log)
+        ios_sdk = subprocess.check_output(
+            ['xcrun', '--show-sdk-path', '--sdk', 'iphonesimulator']
```

Comment:
Does this work on a customer install?

Reply:
I don't see why not, but we will know in ~1hour when the bot checked it.

Reply:
(This is copied&pasted from another test)

Reply:
Ship it then!